README.md

```diff
@@ -209,6 +209,8 @@ options covering different styles. File an issue, or even better, submit a PR.
 * When you run rustfmt, place a file named `rustfmt.toml` or `.rustfmt.toml` in
   target file directory or its parents to override the default settings of
   rustfmt.
+* You can dump the default configuration with
+  `rustfmt --dump-default-config rustfmt.toml` and customize it as needed.
```

Comment:
You don't need `rustfmt.toml` here, it will use that file name as default if you don't specify anything

Reply:
In [bin/rustfmt.rs](https://github.com/rust-lang-nursery/rustfmt/blob/master/src/bin/rustfmt.rs) it does not look like there is a default for the path. Also:
```
~/dev/rust/hello_cargo/src (master #)$ rustfmt --dump-default-config
Argument to option 'dump-default-config' missing
...
~/dev/rust/hello_cargo/src (master #)$ rustfmt --version
0.2.6-nightly ( )
```

Reply:
hmm, there was meant to be a default (I'm sure it was mentioned when we landed this). I guess leave it as is then (if you could file an issue for making a default, that would be great!)

Reply:
Done: #1988
Will wait for it to be closed before updating this PR.

Thanks!